public/styles/header.css

```diff
@@ -0,0 +1,203 @@
+header{
+    /* Header styling if needed */ 
+   }
+   
+   /* #region Mobile Styling */
+   
+       .mobile-menu {
+           display: flex;
+           justify-content: space-between;
+           margin: 5%;
+           padding-bottom: 5%;
+           border-bottom: solid 1px black;
+       }
+       
+       .mobile-logo img {
+           width: 12em;
+       }
+       
+       
+       /* mobile-menu-window */
+       section#mobile-menu-window {
+           display: none;
+           flex-direction: column;
+           position: fixed;
+           inset: 0 auto 0 0;
+           background-color: #f4f4f4;
+           width: 100%;
+           z-index: 4;
+   
+           .menu-exit{
+               position: absolute;
+               left: 0;
+               padding: 1.5em 0em 0em 1.5em;
+           }
+       }
+   
+       
+       #mobile-menu-window ul {
+           display: flex;
+           flex-direction: column;
+           justify-content: center;
+           align-items: center;
+           margin-block: 20%;
+           outline: solid rgb(32, 159, 209);
+           font-size: 2em;
+       }
+       
+       #mobile-menu-window ul li {
+           margin: 0em 0em .25em 0em;
+           font-weight: 600;
+       }
+       
+       #mobile-menu-window ul li a:hover {
+           color: #E84340;
+           transition: .25s;
+       }
+   
+       #mobile-menu-window.showMenu {
+           display: flex;
+           animation: blur-in .15s ease-in;
+       }
+   
+   
+   /* #endregion Mobile Nav Styling */
+
+   /*  #region Desktop styling */
+
+.desktop-menu {
+    font-family: 'Inter', sans-serif;
+}
+
+.top-bar {
+    background-color: black; 
+    color: white;
+    position: relative;
+    width: 100vw;
+    left: 50%;
+    transform: translateX(-50%);
+    padding: 1.25em 0;
+}
+
+
+.desktop-menu picture img {
+    width: 220px;
+}
+
+@supports (display: flex){
```

Comment:
@supports is mostly used when a feature is new or is not widely supported by every browser. but most of the properties you used in this section are commonly used so @supports is not needed